packages/async-operations/tests/test-async_operations.js

```diff
@@ -357,7 +357,7 @@ test('getLambdaEnvironmentVariables returns expected environment variables', (t)
   const vars = getLambdaEnvironmentVariables(t.context.functionConfig);
 
   t.deepEqual(new Set(vars), new Set([
-    { name: 'ES_HOST', value: 'es-host' },
+    { name: 'Timeout', value: 300 },
```

Comment:
Same as above, why this change?

Reply:
I answered this in the other thread, is that sufficient?